### PR TITLE
Aqara Wireless Remote Switch E1 (Single Rocker)

### PR DESCRIPTION
**Pull request by Hongming6**

### CODE REVIEW

File: drivers/SmartThings/zigbee-button/src/test/test_aqara_button.lua [hunk 1]
```diff
@@ -43,13 +58,35 @@ local mock_device_t1 = test.mock_device.build_test_zigbee_device(
...
 zigbee_test_utils.prepare_zigbee_env_info()
 local function test_init()
+  test.mock_device.add_test_device(mock_device_e1)
   test.mock_device.add_test_device(mock_device_t1)
 end
 
 test.set_test_init_function(test_init)
 
+test.register_coroutine_test(
+  "Handle added lifecycle",
```
Comment (greens) [inline drivers/SmartThings/zigbee-button/src/test/test_aqara_button.lua:74]:
```suggestion
  "Handle added lifecycle -- e1",
```

File: drivers/SmartThings/zigbee-button/src/aqara/init.lua [hunk 2]
```diff
@@ -79,15 +82,23 @@ local function device_init(driver, device)
...
+    end
+
     device:emit_event(capabilities.button.supportedButtonValues({"pushed","held","double"}, {visibility = { displayed = false }}))
     device:emit_event(capabilities.button.numberOfButtons({value = 1}))
     device:emit_event(capabilities.button.button.pushed({state_change = false}))
     device:emit_event(capabilities.battery.battery(100))
 end
 
+
+
```
Comment (greens) [inline drivers/SmartThings/zigbee-button/src/aqara/init.lua:101]: nit: extra whitespace

File: drivers/SmartThings/zigbee-button/src/aqara/init.lua [hunk 3]
```diff
@@ -22,15 +22,18 @@ local capabilities = require "st.capabilities"
...
 local MULTISTATE_INPUT_CLUSTER_ID = 0x0012
 local PRESENT_ATTRIBUTE_ID = 0x0055
 
 local FINGERPRINTS = {
-  { mfr = "LUMI", model = "lumi.remote.b1acn02" }
+  { mfr = "LUMI", model = "lumi.remote.b1acn02" },
+  { mfr = "LUMI", model = "lumi.remote.acn003" }
 }
 
+
```
Comment (greens) [inline drivers/SmartThings/zigbee-button/src/aqara/init.lua:36]: nit: extra whitespace

File: drivers/SmartThings/zigbee-button/src/aqara/init.lua [hunk 4]
```diff
@@ -22,15 +22,18 @@ local capabilities = require "st.capabilities"
 local PowerConfiguration = clusters.PowerConfiguration
 local PRIVATE_CLUSTER_ID = 0xFCC0
 local PRIVATE_ATTRIBUTE_ID = 0x0009
+local PRIVATE_SWITCH_MODE_ATTRIBUTE_ID = 0x0125
```
Comment (greens) [inline drivers/SmartThings/zigbee-button/src/aqara/init.lua:25]:
```suggestion
local PRIVATE_ATTRIBUTE_ID_T1 = 0x0009
local PRIVATE_ATTRIBUTE_ID_E1 = 0x0125
```

Comment (Hongming6) [reply drivers/SmartThings/zigbee-button/src/aqara/init.lua]: Thx,has been modified!

File: drivers/SmartThings/zigbee-button/src/test/test_aqara_button.lua [hunk 5]
```diff
@@ -1,3 +1,4 @@
+---@diagnostic disable: param-type-mismatch, missing-parameter
```
Comment (greens) [inline drivers/SmartThings/zigbee-button/src/test/test_aqara_button.lua:1]: remove

File: drivers/SmartThings/zigbee-button/src/aqara/init.lua [hunk 6]
```diff
@@ -79,9 +81,15 @@ local function device_init(driver, device)
...
-    device:send(cluster_base.write_manufacturer_specific_attribute(device,
-                PRIVATE_CLUSTER_ID, PRIVATE_ATTRIBUTE_ID, MFG_CODE, data_types.Uint8, 1))
+local function added_handler(self, device)
+    if device:get_model() == "lumi.remote.b1acn02" then
+      device:send(cluster_base.write_manufacturer_specific_attribute(device,
+      PRIVATE_CLUSTER_ID, PRIVATE_ATTRIBUTE_ID_T1, MFG_CODE, data_types.Uint8, 1))
+    elseif device:get_model() == "lumi.remote.acn003" then
+      device:send(cluster_base.write_manufacturer_specific_attribute(device,
+      PRIVATE_CLUSTER_ID, PRIVATE_ATTRIBUTE_ID_E1, MFG_CODE, data_types.Uint8, 2))
+    end
```
Comment (greens) [inline drivers/SmartThings/zigbee-button/src/aqara/init.lua:91]: a comment indicating what these both do would be helpful

Comment (Hongming6) [reply drivers/SmartThings/zigbee-button/src/aqara/init.lua:91]: ok,when the wireless switch T1 accesses the network, the gateway sends private attribute 0009 to make the device no longer distinguish between the standard gateway and the aqara gateway. When wireless switch E1 is connected to the network, the gateway delivers private attribute 0125 to enable the device to send double-click and long-press packets. Wireless switch T1 supports double and long press by default, while wireless switch E1 does not distinguish between standard gateway and aqara gateway by default. This is the point where they issue different instructions when they enter the network.

**Review by wkhenon — APPROVED** [review]

**Review by greens — APPROVED** [review]

**Comment by greens** [comment]
```
Checking drivers/SmartThings/zigbee-button/src/aqara/init.lua 4 warnings

    drivers/SmartThings/zigbee-button/src/aqara/init.lua:92:75: (W614) trailing whitespace in a comment
    drivers/SmartThings/zigbee-button/src/aqara/init.lua:93:71: (W614) trailing whitespace in a comment
    drivers/SmartThings/zigbee-button/src/aqara/init.lua:94:59: (W614) trailing whitespace in a comment
    drivers/SmartThings/zigbee-button/src/aqara/init.lua:95:78: (W614) trailing whitespace in a comment
```

**Pull request merged**